src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -479,12 +485,32 @@ redeemADA :: WalletWebMode ssc m => SendActions m -> CWalletRedeem -> m CTx
 redeemADA sendActions CWalletRedeem {..} = do
     let base64rify = T.replace "-" "+" . T.replace "_" "/"
```

Comment:
I think i would go with `decodeUrl bs <|> decode bs` instead next time (seems to be more error-prone). Though I see no real reason to change anything.